### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mickeygousset**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,7 @@ git reset --hard <the latest version>
 git push --force
 
 
+
+
```
Comment (mickeygousset) [inline README.md:13]:
```suggestion
asdfasdfasdfasdfasdasdf
```

File: README.md [hunk 2]
```diff
@@ -9,3 +9,7 @@ git reset --hard <the latest version>
 git push --force
 
 
+
```
Comment (mickeygousset) [inline README.md:12]: asdfasdfsadsdf

Comment (mickeygousset) [inline README.md:12]:
```suggestion
aaaaaaaaaaaaaaaaaaaaaaaaaaa
```

File: README.md [hunk 3]
```diff
@@ -9,3 +9,7 @@ git reset --hard <the latest version>
 git push --force
```
Comment (mickeygousset) [inline README.md:10]: asdfsdf

File: README.md [hunk 4]
```diff
@@ -9,3 +9,7 @@ git reset --hard <the latest version>
...
```
Comment (mickey-octodemo) [inline README.md:9]:
```suggestion
git push --forceaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

**Review by mickey-octodemo — APPROVED** [review]
asdf

**Pull request closed without merging**